src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolver.java

```diff
@@ -50,7 +51,10 @@ public Class<DeleteOneCommand> getCommandClass() {
   private FindOperation getFindOperation(CommandContext commandContext, DeleteOneCommand command) {
     List<DBFilterBase> filters = resolve(commandContext, command);
     final SortClause sortClause = command.sortClause();
-
+    if (sortClause != null && commandContext.embeddingService() != null) {
+      new VectorizeData(commandContext.embeddingService(), objectMapper.getNodeFactory())
```

Comment:
as per earlier comments, I think `VectorizeData` name could be improved, as it's more like DataVectorizer or something.